kafka/src/kafka/impl/error_buffer.cpp

```diff
@@ -18,6 +18,12 @@ void PrintErrorAndThrow(std::string_view failed_action, const ErrorBuffer& err_b
     throw std::runtime_error{full_error};
 }
 
+void PrintErrorAndThrow(std::string_view failed_action, std::string_view reason) {
```

Comment:
It is not good solution for runtime methods, please add custom exceptions

Reply:
done